### ISSUE TITLE
pickupManage: fill aria-labels

### ISSUE THREAD

**Issue by tiltec** [yunity/foodsaving-frontend#461]
There's a lot of warnings due to missing aria-labels. Need to fill those using the translate filter.

I suggest to use [one-time bindings](https://docs.angularjs.org/guide/expression) to improve performance (watches are slow). The removes the possibility to switch languages on the fly, but a page reload does the job.
